src/set.js

```diff
@@ -0,0 +1,99 @@
+export default class Set {
+  constructor() {
+    this.length = 0
+    this.elements = []
+  }
+
+  add = (value) => {
```

Comment:
Consider consistent style when defining methods in classes (your constructor, above, is written in a different style).  I personally prefer:
```js
add( value ) {
  // etc.
}
```